README.md

```diff
@@ -135,17 +135,17 @@ After the checkup Job had completed, the results are made available at the user-
 kubectl get configmap dpdk-checkup-config -n <target-namespace> -o yaml
 ```
 
-| Key                                        | Description                                                       | Remarks  |
-|--------------------------------------------|-------------------------------------------------------------------|----------|
-| status.succeeded                           | Has the checkup succeeded                                         |          |
-| status.failureReason                       | Failure reason in case of a failure                               |          |
-| status.startTimestamp                      | Checkup start timestamp                                           | RFC 3339 |
-| status.completionTimestamp                 | Checkup completion timestamp                                      | RFC 3339 |
-| status.result.trafficGenSentPackets        | Number of packets sent from the traffic generator                 |          |
-| status.result.trafficGenOutputErrorPackets | Indicates error sending packets from traffic generator            |          |
-| status.result.trafficGenInputErrorPackets  | Indicates error receiving packets to traffic generator            |          |
-| status.result.trafficGenActualNodeName     | Node name on which the traffic generator VM was scheduled         |          |
-| status.result.vmUnderTestActualNodeName    | Node name on which the VM under test was scheduled                |          |
-| status.result.vmUnderTestReceivedPackets   | Number of packets received on the VM under test                   |          |
-| status.result.vmUnderTestRxDroppedPackets  | Indicates ingress packets that were dropped from DPDK application |          |
-| status.result.vmUnderTestTxDroppedPackets  | Indicates egress packets were dropped from the DPDK application   |          |
+| Key                                        | Description                                                            | Remarks  |
+|--------------------------------------------|------------------------------------------------------------------------|----------|
+| status.succeeded                           | Specifies if the checkup is successful (`true`) or not (`false`)       |          |
+| status.failureReason                       | The reason for failure if the checkup fails                            |          |
+| status.startTimestamp                      | The time when the checkup started                                      | RFC 3339 |
+| status.completionTimestamp                 | The time when the checkup is complete                                  | RFC 3339 |
```

Comment:
`startTimestamp` uses past tense "started" while - `completionTimestamp` uses present tense "is complete". could you align?